src/gameplay/game.ts

```diff
@@ -566,6 +570,9 @@ class Game extends Room {
     if (this.disableVoteHistory) {
       this.sendText('The game has vote history disabled.', 'gameplay-text');
     }
+    if (this.enableSinadMode) {
+      this.sendText('The game has Sinad Mode enabled.', 'gameplay-text');
+    }
```

Comment:
Can you also check that enableSinadMode can only be true if it's 6P here? 
https://github.com/vck3000/ProAvalon/blob/master/src/gameplay/game.ts#L362-L378

This way if the host misconfigures the game, the server will tell them when they try to start the game.

Reply:
My [current solution](<https://github.com/vck3000/ProAvalon/pull/713/files#diff-25e21f94af91cda96dca3d6af6ff4bb78a52fd8c85cfcdbd9418471233554e77R622-R6331>) to this.

I wanted to make this check inside `static checkOptions(options: string[])`, but I wasn't sure because it looks like the `options` array only has Role enum members, and I *think* `startGame(options: string[]) ` [relies](<https://github.com/vck3000/ProAvalon/blob/master/src/gameplay/game.ts#L445-L459>) on this. Alternatively, I could try making `checkOptions` non-static to access `this.enableSinadMode` directly, but looks tricky.

Reply:
I see. Feel free to make that non-static so you can put it together. 

The design of Room and Game is pretty poor. Ideally these kinds of checks are done within the Game.